src/emulator/gui/src/profiles_manager_dialog.cpp

```diff
@@ -0,0 +1,189 @@
+                host.cfg.user_id += (int)host.cfg.online_id.size() - host.cfg.user_id;
+                gui.online_id.clear();
+            }
+            ImGui::CloseCurrentPopup();
+        }
+        ImGui::EndPopup();
+    } else
+        ImGui::PopStyleColor();
+
+    ImGui::PushItemWidth(300);
+    if (ImGui::ListBoxHeader("##online_id_list", (int)host.cfg.online_id.size(), 3)) {
+        for (auto i = 0; i < host.cfg.online_id.size(); i++) {
+            ImGui::Text("User ID: %02d, Online ID: %s", i, host.cfg.online_id[i].c_str());
+        }
+        ImGui::ListBoxFooter();
+    }
+    ImGui::PopItemWidth();
+    ImGui::Spacing();
+    if (host.cfg.online_id.size() > 1) {
+        ImGui::Combo("Select your User", &host.cfg.user_id,
+            [](void *vec, int idx, const char **list_online_id) {
+                auto &vector = *static_cast<std::vector<std::string> *>(vec);
+                if (idx < 0 || idx >= static_cast<int>(vector.size())) {
+                    return false;
+                }
+                *list_online_id = vector.at(idx).c_str();
+                return true;
+            },
+            static_cast<void *>(&host.cfg.online_id), (int)host.cfg.online_id.size());
+        ImGui::Spacing();
+        if (ImGui::Button("Delete Selected User"))
+            ImGui::OpenPopup("Delete User");
+        if (ImGui::IsItemHovered())
+            ImGui::SetTooltip("Obligatory keep one Online ID in same time.");
+        ImGui::PushStyleColor(ImGuiCol_Text, GUI_COLOR_TEXT_MENUBAR_OPTIONS);
+        if (ImGui::BeginPopupModal("Delete User", NULL, ImGuiWindowFlags_AlwaysAutoResize)) {
+            ImGui::PopStyleColor();
+            auto delete_online_id = std::find(host.cfg.online_id.begin(), host.cfg.online_id.end(), host.cfg.online_id[host.cfg.user_id]);
+            const auto user_id_path = fs::path(host.pref_path) / "ux0/user" / host.io.user_id;
+            if (fs::exists(user_id_path)) {
+                ImGui::Text("Do you want also delete folder for this user ?\nUser ID: %s, Online ID: %s", host.io.user_id, host.cfg.online_id[host.cfg.user_id]);
+                ImGui::Separator();
+                if (ImGui::Button("Yes", ImVec2(100, 0))) {
+                    fs::remove_all(user_id_path);
+                    host.cfg.online_id.erase(delete_online_id);
+                    ImGui::CloseCurrentPopup();
+                }
+                ImGui::SameLine();
+                if (ImGui::Button("No", ImVec2(100, 0))) {
+                    host.cfg.online_id.erase(delete_online_id);
+                    ImGui::CloseCurrentPopup();
+                }
+                ImGui::SameLine();
+                if (ImGui::Button("Cancel", ImVec2(100, 0))) {
+                    ImGui::CloseCurrentPopup();
+                }
+            } else {
+                ImGui::Text("Do you want really delete this user ?\nUser ID: %s, Online ID: %s", host.io.user_id, host.cfg.online_id[host.cfg.user_id]);
```

Comment:
"Do you **really** want **to** delete this user?"

Reply:
Ah thx ^^